### PR TITLE
Persian (Farsi) FluxBB v1.5.9

### PR DESCRIPTION
**Pull request by abdulhalim**
Persian Language for FluxBB version 1.5.9

### CODE REVIEW

File: Farsi/admin_forums.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+'Confirm delete head'		=>	'تائید حذف انجمن',
+'Confirm delete subhead'	=>	'بسیار مهم ! قبل از حذف مطالعه شود',
+'Confirm delete info'		=>	'آیا مطمئن هستید که میخواهید انجمن <strong>%s</strong> را حذف کنید؟',
+'Confirm delete warn'		=>	'<strong>هشدار!</strong> با حذف یک انجمن تمام ارسال ها (درصورت وجود) در این انجمن نیز حذف خواهند شد!',
+
+// Detailed edit page
+'Edit forum head'			=>	'ویرایش انجمن',
+'Edit details subhead'		=>	'ویرایش جزئیات انجمن',
+'Forum name label'			=>	'نام انجمن',
+'Forum description label'	=>	'توضیحات (HTML)',
```
Comment (oldskool) [inline Farsi/admin_forums.php]: Shouldn't these be the letters H, T, M and L in Farsi? Or is that very unusual?

Comment (abdulhalim) [inline Farsi/admin_forums.php]: Do you mean HTML ? It depends on translators , i prefer to use HTML instead of اچ تی ام ال

Comment (oldskool) [inline Farsi/admin_forums.php]: Yeah, I meant the HTML part. But other titles and abbreviations are also still in English, so I guess it makes sense to keep it that way.

Comment (abdulhalim) [inline Farsi/admin_forums.php]: Yes , i think this is the best way , coz something can't to be translated , especially in such thing .

**Comment by franzliedke** [comment]
Please ignore my ignorance. This is totally different than the [existing Persian language pack](https://github.com/fluxbb/langs/tree/ea587bbd063e33dd10422777528847796b54085b/Persian), right?

**Comment by abdulhalim** [comment]
Yes , this is for latest version of FluxBB , also i translated from scratch , another thing , there is no difference between Persian and Farsi ( both are for Iranian language ) . i selected Farsi for not conflicted with another Persian language

**Comment by franzliedke** [comment]
Hmm, I'd prefer if you could combine the two, or improve the other one, so that we don't have two versions for one language.

**Comment by abdulhalim** [comment]
Could you do it yourself please ?? i am not familiar with GitHub .

**Comment by franzliedke** [comment]
I cannot understand the differences between your version and the existing one, though.

Here's my suggestion: You can go through [these files](https://github.com/fluxbb/langs/tree/master/Persian) one-by-one, and use GitHub's edit function (the little pencil icon on the top right when viewing a file) to send pull requests, file-by-file, with your changes. How does that sound?

**Comment by franzliedke** [comment]
Closing because of inactivity.

**Pull request closed without merging**